src/hooks/useMetrika.ts

```diff
@@ -3,6 +3,10 @@ import {MetrikaContext} from '../context/metrikaContext';
 import {ButtonPixel, MetrikaGoal, PixelEventType, PixelEvent} from '../models';
 import {isNewMetrikaFormat} from '../models/guards';
 
+// eslint-disable-next-line valid-jsdoc
```

Comment:
why do we need this disabling?

Reply:
because of @deprecated 
without the row @deprecated comment requires me to fill out all the fields (such as `return` type etc.) 
But it is not important for us in context of PR